nixos/modules/services/misc/rkvm.nix

```
@@ -0,0 +1,168 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.rkvm;
+  toml = pkgs.formats.toml { };
+in
+{
+  meta.maintainers = with maintainers; [ ckie ];
+
+  options.services.rkvm = {
+    enable = mkOption {
+      default = cfg.server.enable || cfg.client.enable;
+      defaultText = literalExpression "cfg.server.enable || cfg.client.enable";
+      type = types.bool;
+      description = lib.mdDoc ''
```

for consistency's sake
```suggestion
      description = mdDoc ''
```

---

thanks!